teamsnap_rb.gemspec

```diff
@@ -1,30 +1,32 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'teamsnap_rb/version'
+require "teamsnap/version"
 
 Gem::Specification.new do |spec|
   spec.name          = "teamsnap_rb"
-  spec.version       = TeamsnapRb::VERSION
-  spec.authors       = ["Party Chicken"]
+  spec.version       = TeamSnap::VERSION
+  spec.authors       = ["Shane Emmons", "Kyle Ries"]
   spec.email         = ["api@teamsnap.com"]
   spec.summary       = %q{A gem interact with TeamSnap's API}
-  spec.description   = %q{}
-  spec.homepage      = "http://www.teamsnap.com/api"
+  spec.description   = %q{A gem interact with TeamSnap's API}
```

Comment:
fixed


*Shane Emmons* | Director of Technology and Principal Architect
TeamSnap - The easiest way to manage your team
(810) 834-9542 | shane@teamsnap.com | What is TeamSnap?
<http://www.youtube.com/watch?v=fx0vRXYGijU>

On Mon, Feb 16, 2015 at 12:25 PM, Kyle <notifications@github.com> wrote:

> In teamsnap_rb.gemspec
> <https://github.com/teamsnap/teamsnap_rb/pull/39#discussion_r24764355>:
>
> >    spec.email         = ["api@teamsnap.com"]
> >    spec.summary       = %q{A gem interact with TeamSnap's API}
> > -  spec.description   = %q{}
> > -  spec.homepage      = "http://www.teamsnap.com/api"
> > +  spec.description   = %q{A gem interact with TeamSnap's API}
>
> Small typo - missing "to"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/teamsnap/teamsnap_rb/pull/39/files#r24764355>.
>